src/huggingface_hub/inference/_client.py

```diff
@@ -1990,3 +1991,50 @@ def get_model_status(self, model: Optional[str] = None) -> ModelStatus:
             compute_type=response_data["compute_type"],
             framework=response_data["framework"],
         )
+
+    def audio_to_audio(
+        self,
+        audio: ContentT,
+        *,
+        model: Optional[str] = None,
+    ) -> List[bytes]:
```

Comment:
Could you define an `AudioToAudioOutput` type [in this file](https://github.com/huggingface/huggingface_hub/blob/main/src/huggingface_hub/inference/_types.py) which would be a typed dict with label/content_type/blob in it? Once defined, the annotation type for `audio_to_audio` method should be `List[AudioToAudioOutput]`.